SVN: do not ship .map or .scss files in the .org releases.

jeherve opened the pull request:
Fixes #10626

#### Changes proposed in this Pull Request:

Do not ship .map or .scss files in the .org releases.

#### Testing instructions:

* When building a branch for release, make sure no .scss or .map files are included.

#### Proposed changelog entry for your changes:
 
* None.

jeherve committed:
SVN: do not ship .map or .scss files in the .org releases.

Fixes #10626

brbrr reviewed (dismissed):
Looks good :shipit:

jeherve committed:
SVN: do not use * to ignore a full directory

SVN ignore only takes files into account:
http://svnbook.red-bean.com/en/1.7/svn-book.html#svn.advanced.props.special.ignore

jeherve committed:
gitignore: we do not need .map in the release files.

oskosk reviewed (approved):
LGTM!

jeherve commented:
Cherry-picked to `branch-6.8` in e785eac812d26e6941594c491b944dc3fc115bda
